src/custom/pages/Claim/InvestmentFlow/InvestSummaryRow.tsx

```diff
@@ -0,0 +1,73 @@
+import { ClaimType } from 'state/claim/hooks'
+import { calculatePercentage } from 'state/claim/hooks/utils'
+import { TokenLogo } from 'pages/Claim/styled'
+import { ClaimWithInvestmentData } from 'pages/Claim/types'
+import CowProtocolLogo from 'components/CowProtocolLogo'
+import { formatSmart } from 'utils/format'
+import { AMOUNT_PRECISION } from 'constants/index'
+
+export type Props = { claim: ClaimWithInvestmentData }
+
+export function InvestSummaryRow(props: Props): JSX.Element | null {
+  const { claim } = props
+
+  const { isFree, type, price, currencyAmount, vCowAmount, cost, investmentCost } = claim
+
+  const symbol = isFree ? '' : (currencyAmount?.currency?.symbol as string)
+
+  const formattedCost =
+    formatSmart(investmentCost, AMOUNT_PRECISION, { thousandSeparator: true, isLocaleAware: true }) || '0'
+  const percentage = investmentCost && cost && calculatePercentage(investmentCost, cost)
+
+  return (
+    <tr>
+      <td>
+        {isFree ? (
+          <b>{ClaimType[type]}</b>
+        ) : (
+          <>
+            <TokenLogo symbol={symbol} size={32} />
+            <CowProtocolLogo size={32} />
+            <span>
+              <b>Buy vCOW</b>
+              <i>with {symbol}</i>
+            </span>
+          </>
+        )}
+      </td>
+
+      <td>
+        {!isFree && (
+          <span>
+            <b>Investment amount:</b>{' '}
+            <i>
+              {formattedCost} {symbol} ({percentage}% of available investing opportunity)
```

Comment:
@michelbio, should we style differently the non 100% cases? 
not investing 100% is something that cannot be undone. 

I would even omit the percentage if it's 100%.
This could be done in another PR

Reply:
Yes we can review.